webpack.common.js

```diff
@@ -92,7 +92,6 @@ function generateHTML(){
   pages.unshift('index.html');
   const htmlPages=pages.filter(page => path.extname(page)=='.html');
   return htmlPages.map(item => {
-    const name=item.split((__dirname+'/src/pages/')).at(-1).split('.')[0];
```

Comment:
You removed the `console.log` statement but also removed this line which you shouldn't have since it also breaks the build.